src/main/java/com/prgrms/jpa/jpa1/domain/Order.java

```diff
@@ -0,0 +1,98 @@
+package com.prgrms.jpa.jpa1.domain;
+
+import jakarta.persistence.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "orders")
+public class Order extends BaseEntity{
+    @Id
+    @Column(name = "id")
+    private String uuid;
+
+    @Lob
+    private String memo;
+
+    @Enumerated(value = EnumType.STRING)
+    private OrderStatus orderStatus;
+
+    @Column(name = "order_datetime")
+    private LocalDateTime orderDateTime;
+
+    @Column(name = "member_id", insertable = false, updatable = false)
+    private Long memberId;
+    // order.setMemberId 했을 때 실행되지 않도록 설정
+
+    @ManyToOne
+    @JoinColumn(name = "member_id", referencedColumnName = "id")
+    private Member member;
+
+    @OneToMany(mappedBy = "order")
+    private List<OrderItem> orderItems = new ArrayList<>();
+
+
+    public String getUuid() {
+        return uuid;
+    }
+
+    public void setUuid(String uuid) {
+        this.uuid = uuid;
+    }
+
+    public String getMemo() {
+        return memo;
+    }
+
+    public void setMemo(String memo) {
+        this.memo = memo;
+    }
+
+    public OrderStatus getOrderStatus() {
+        return orderStatus;
+    }
+
+    public void setOrderStatus(OrderStatus orderStatus) {
+        this.orderStatus = orderStatus;
+    }
+
+    public LocalDateTime getOrderDateTime() {
+        return orderDateTime;
+    }
+
+    public void setOrderDateTime(LocalDateTime orderDateTime) {
+        this.orderDateTime = orderDateTime;
+    }
+
+    public Long getMemberId() {
+        return memberId;
+    }
+
+    public void setMemberId(Long memberId) {
+        this.memberId = memberId;
+    }
+
+    public Member getMember() {
+        return member;
+    }
+
+    public void setMember(Member member) { // 연관관계 편의 메소드
+        if(Objects.nonNull(this.member)){
+            member.getOrders().remove(this);
+        }
+
+        this.member = member;
+        member.getOrders().add(this);
```

Comment:
순서가 중요하지않다면 Set 컬렉션을 사용해봐도 좋겠네요.

```suggestion
        this.member = member;
        member.getOrders().add(this);
```